public/locales/ru/translations.json

```diff
@@ -0,0 +1,183 @@
+{
+  "language name": "Русский",
+
+  "home": "Главная",
+  "recent": "Недавняя Активность",
+  "no followers": "Ни одного подписчика",
+  "latest followers": "Последние Подписавшиеся",
+  "connect wallet to view": "Подключите кошелек, чтобы увидеть",
+  "phrase": "Все о тех, кого вы знаете. 😜",
+  "social graph first": "Социальная сеть",
```

Comment:
Social graphs is a phrase that is not used in everyday life in Russian. Instead, "social network" is used. The concept is broader, but very common.

Reply:
Got it. In the Russian marketing school, there is a notion that with Russians it is always worth using the most general concepts. However, if you also want to educate people, the term social graph will do just fine.